### PR TITLE
Hello Microverse project pull request

### PR DESCRIPTION
**Pull request by lengushuru**
this is a setup of the Hello Microverse Project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
```
Comment (leonardodiasb) [inline index.html:7]: - [ ] I see that you created a `styles.css` file but it was not imported into your HTML file, so the styling effects are not being applied to your project. Please import your CSS file so the style can be applied.

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @lengushuru, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - Linters set up correctly. ✔️
 - Correct use of Github flow. ✔️ 

## Aspects to improve ♻️

- [x] Check the comments under the review.
- [x] Your README file is a copy of the template Microverse provides. Please, customize the information inside the templated so the README represents this project and it looks more professional. Add information on the topics that are blank, change the information about the author, the issues link, the project title, and description, etc.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by fherrerao — CHANGES_REQUESTED** [review]
Hi @lengushuru , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

So Close Snl GIF by Saturday Night Live
 ## Highlights
✅ Greate use of GitHub flow.
✅ Good commit messages.
✅ No linter errors.

 ## Required Changes ♻️

- [ ] Please, as the previous reviewer says, you used the readme template but did not edit it with information related to your project (author, used technologies, etc. ). Please update the file and request a new review.

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle: 

Hi @lengushuru, 

Great work on making the changes requested by the previous reviewers 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
 -  No linter errors. :heavy_check_mark: 
 -   Correct GitHub  Flow. :heavy_check_mark: 



 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

- [ ] You've not still implemented the request from one of the previous reviewers as regard your README file, Kindly attend to that and ask for another review. Thank you. :handshake:

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: blue;
+}
```
[the remark this answers is not in the document]
Comment (lengushuru) [reply styles.css:3]: @Qoosim should I request an extra review or should I make the changes you have pointed out only

**Comment by lengushuru** [comment]
I have tried as much as I could to address the issues pointed out by the reviewers. please provide detailed issues that need to be addressed in the README.md file.

File: styles.css (hunk 2 above)
[the remark this answers is not in the document]
Comment (fobadara) [reply styles.css]: Sorry about that. Your project should have been approved.
But I think you should delete the empty subheadings like setup and install

**Review by fobadara — APPROVED** [review]
Hi @lengushuru, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✅ Passing linters
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**